sys/net/routing/aodvv2/aodv.c

```diff
@@ -477,6 +463,32 @@ static void _write_packet(struct rfc5444_writer *wr __attribute__ ((unused)),
     AODV_DEBUG("%d bytes sent.\n", bytes_sent);
 }
 
+/* Print the json representation of a sent packet to stdout for debugging */
+static void print_json_pkt_sent(struct writer_target *wt)
+{
+    // note: what if the content at wt has changed until this is printed? memcpy the entire thing?
```

Comment:
This seems like a good thing to do, especially considering that this will mostly be run on native, right?